src/components/site/BreadCrumbLinkItem.js

```diff
@@ -4,9 +4,9 @@ import PropTypes from "prop-types";
 const BreadcrumbLinkItem = props => {
   const { link, text } = props;
   return (
-    <li className="dg_breadcrumb">
-      <a href={link}>{text}</a>
-    </li>
+    <a className="breadcrumb" href={link}>
```

Comment:
you should add proptypes to this component.

Reply:
Refactored this item to accept classNames, so the required `.breadcrumb-last` class that SE creates can be added to the example in the components library. Let me know if this looks ok. I also updated other files that had dependencies for the BreadcrumbLastItem.js and BreadcrumbItem.js for that matter, since they are not necessary when using the refactored version of this component.